server/controllers/RequestController.js

```diff
@@ -0,0 +1,32 @@
+import { requests } from '../dummydatabase/dummydatabase';
+/**
+  * @class RequestController
+  * @description CRUD operations on Requests
+  */
+class RequestController {
+  /**
+        * @static
+        * @param {object} req - The request payload sent to the router
+        * @param {object} res - The response payload sent back from the controller
+        * @returns {object} - status Message and the particular request by id.
+        * @description This method gets a request by id in maintenance tracker
+        * @memberOf RequestController
+        */
+  static getRequestById(req, res) {
+    const id = req.params.requestId;
```

Comment:
Cover edge cases do you know what happens when I try to typecast a string to a number?

Reply:
do you mean parseInt() ? I am thinking that method might be porous?  my app might now listen to     URLs such as https://api/v1/users/requests/339ahvfish